test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperSelectTest.cs

```diff
@@ -1299,6 +1301,30 @@ public void GetEnumSelectListTEnum_ThrowsWithNonEnum()
         }
 
         [Fact]
+        public void GetEnumSelectListTEnum_LocalizedEnum()
```

Comment:
Do not use "Localized" as if it means "using an `IStringLocalizer`". Change test name to describe the specific scenario.